docs/source/glossary.rst

```diff
@@ -0,0 +1,164 @@
+    Setting or value that is stored locally, rather than on the blockchain.
+
+  On-chain setting
+    Setting or value that is stored on the blockchain (also referred to as
+    "in state") so that all participants on the network can access that
+    information.
+
+  Permissioned network
+    Private network of Sawtooth nodes. The blockchain stores the settings that
+    specify permissions, such as roles and identities, so that all participants
+    in the network can access this information.
+
+  PoET
+    Proof of Elapsed Time, a Nakamoto-style consensus algorithm that is designed
+    to support large networks. PoET relies on a trusted execution environment
+    in |Intel (R)| CPUs called Software Guard Extensions (SGX).  For more
+    information, see :doc:`architecture/poet`.
+
+  REST API
+    Component that adapts communication between the client application and
+    the validator. Sawtooth includes a REST API that adapts validator
+    communication to standard HTTP/JSON (and vice versa),
+    but application developers could provide a custom REST API.
+
+  Sawtooth core
+    Central Sawtooth software that is responsible for message handling,
+    block validation and publishing, consensus, and global state management.
+    The Sawtooth architecture separates these core functions from
+    application-specific business logic, which is is handled by
+    transaction families.
+
+  Sawtooth network
+    Peer-to-peer network of nodes running a validator (and associated
+    components) that are working on the same blockchain.
+
+  Settings
+    Sample transaction family that provides a reference implementation for
+    storing on-chain configuration settings. For more information, see
+    :doc:`transaction_family_specifications/settings_transaction_family`.
+
+  State
+    See :term:`Global state`.
+
+  State delta
+    Result of a single change for a specific address in global state.
+
+  State delta subscriber
+    Client framework that subscribes to a validator for state deltas (changes)
+    for a specific set of transaction families. Usually, an application
+    subscribes to state deltas for the purpose of off-chain storage or action,
+    such as handling the failure of a transaction appropriately.
+
+  Transaction
+    A single state change for the blockchain. Each transaction is put into
+    a Batch, either alone or with other related transactions, then sent to the
+    validator for processing.  For more information, see
+    :doc:`architecture/transactions_and_batches`.
+
+  Transaction family
+    Application-specific business logic that defines a set of operations or
+    transaction types that are allowed on the blockchain. A transaction family
+    implements a data model and transaction language for an application.
+    Sawtooth transaction families separate the transaction rules and
+    content from the Sawtooth core functionality.
+
+    Sawtooth includes example transaction family SDKs in several languages,
```

Comment:
example transaction family SDKs -> example transaction families

The SDKs are something different in my estimation.

Reply:
SDKs should probably be removed.